### PR TITLE
Add products controller

### PR DESCRIPTION
**Pull request by mellava**
Added '/products' ,'/products/:id', '/products/:id/styles', '/products/:id/related'

Screenshot 2023-05-04 at 5 50 26 PM

Screenshot 2023-05-04 at 5 47 15 PM

Screenshot 2023-05-04 at 5 48 27 PM

Screenshot 2023-05-04 at 5 49 17 PM

### CODE REVIEW

File: server/controllers/ProductController.js [hunk 1]
```diff
@@ -1,8 +1,53 @@
+require('dotenv').config();
 const express = require('express');
+const axios = require('axios');
 const router = express.Router();
 
+const API_URL = process.env.API_ENDPOINT;
```
Comment (mellava) [inline server/controllers/ProductController.js:6]: API_ENDPOINT='https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp'

File: server/controllers/ProductController.js [hunk 2]
```diff
@@ -1,8 +1,53 @@
...
+})
+
+router.get('/:id', (req, res) => {
+  axios.get(`${API_URL}/products/${req.params.id}`, {headers : {'Authorization': API_KEY}})
+  .then((response) => {
+    res.send(response.data)
+  })
+  .catch((error) => {
+    console.error(error)
+    res.sendStatus(error.response.status)
```
Comment (mellava) [inline server/controllers/ProductController.js:27]: Allows for dynamic error status

**Review by SashaGordin — APPROVED** [review]
Have never used error.response.status but code looks clean

**Review by SashaGordin — APPROVED** [review]
resolved conversations

**Pull request merged**